packages/client/src/game-core/constants.ts

```diff
@@ -1,5 +1,6 @@
 export const CanvasWidth = 500
-export const CanvasHeight = window.innerHeight - 150
+export const CanvasHeight =
+    typeof window !== 'undefined' ? window.innerHeight - 150 : 0
```

Comment:
Не особо повлияет на что-то, но пусть хотя бы квадрат будет на всякий случай)

```suggestion
    typeof window !== 'undefined' ? window.innerHeight - 150 : CanvasWidth
```